CHANGELOG.md

```diff
@@ -3,38 +3,38 @@ Changelog
 
 Version 0.5
 -----------
-* ADD: basic use of binary messages for faster coordinate data transfer
-* ADD: updated `NGL` code, (e.g. orthographic camera, anti-aliasing, much faster rendering, ...)
-* ADD: `delay`
-* ADD: `show_url`
-* ADD: `hide` and `show_only` methods
-* ADD: `show_rdkit`
-* ADD: `clear` for clearing representations
-* ADD: `add_unitcell` representation
-* ADD: attribute `view.trajectory_0`, `view.trajectory_1`, `view.component_0`, ...
-* ADD: specify `component` index in `add_representations`, `clear_representations`
-* ADD: drag and drop file to widget
-* ADD: `player`
-* ADD: `camera` (`perspective` and `orthographic`)
-* ADD: `background`
-* ADD: `orientation` for syncing two widget cameras
-* ADD: viewing many trajectories, `add_trajectory`, `add_component`
-* ADD: `download_image`
-* ADD: `render_image`
-* ADD: `center_view`
-* ADD: atom selection by array
-* ADD: shortcut for add_representation (add_cartoon, add_rope, ...)
-* ADD: `ParmEdTrajectory` adaptor and `show_parmed`
-* ENH: `add_trajectory` can accept `pytraj.Trajectory`, `mdtraj.Trajectory`, `MDAnalysis.Universe`, `parmed.Structure`
-* ENH: add_representations can accept both snake and CamelKeyword
+
+* ADD: `view.delay` property to adjust trajectory playback
+* ADD: `show_url` function for loading & viewing from an url
+* ADD: `view.hide()` and `view.show_only()` convenience methods
+* ADD: `show_rdkit` function to view rdkit Mol objects
+* ADD: `view.clear()` for clearing all representations
```

Comment:
Actually `view.clear(component=index)`. :D (default index=0)

which is same as `view.add_representations(component=0)`